backstory/src/main/java/com/google/sps/perspective/PerspectiveStoryAnalysisManager.java

```diff
@@ -110,22 +110,14 @@ public StoryDecision generateDecision(String story) throws NoAppropriateStoryExc
    * as a PerspectiveDecision object (for demo purposes).
    *
    * @param story The story to be analyzed
-   * @return An object describing the recommendation resulting from the analysis.
-   * @throws NoAppropriateStoryException if story is not considered appropriate
+   * @return An object describing the recommendation resulting from the analysis. If the
+   *    the PerspectiveDecision story field is null, it's not an appropriate story.
    */
-  public PerspectiveDecision generatePerspectiveDecision(String story)
-      throws NoAppropriateStoryException {
+  public PerspectiveDecision generatePerspectiveDecision(String story) {
     PerspectiveAPIClient apiClient = new PerspectiveAPIClient(perspectiveAPI);
     PerspectiveValues storyValues = apiClient.analyze(Arrays.asList(REQUESTED_ATTRIBUTES), story);
     boolean isStoryAppropriate = ContentDecisions.makeDecision(storyValues);
 
-    // if content decisions returns that it's appropriate
-    // then return a StoryDecision object with this story
-    if (isStoryAppropriate) {
-      return new PerspectiveDecision(story, storyValues);
-    }
-
-    // otherwise throw the NoAppropriateStoryException
-    throw new NoAppropriateStoryException("The story passed in was not appropriate.");
```

Comment:
Can the NoAppropriateStoryException file be deleted too?

Reply:
This is only updated for PerspectiveDecision, which is the version of StoryDecision that the Perspective demo uses (but NOT the BackstoryServlet) as such NoAppropriateStoryException is still necessary because it's thrown in StoryDecision.

Reply:
Done

Reply:
Thanks for explaining.